tools/travis-build-tests.sh

```diff
@@ -12,5 +12,22 @@ export TRAVIS_COMMIT_MESSAGE=no_message
 
 if [ "$BUILD_TESTS" != "false" ]; then
     cd ${BASE}/big_tests && make prepare
+
+    # Copy data files, so tests, compiled by rebar still
+    # can find the data.
+    #
+    # Common Tests use filename from beam file to figure out data_dir.
+    # data_dir for rebar compiled files would be somewhere in _build directory,
+    # not where we would expect it to be.
+    #
+    # Used with SKIP_AUTO_COMPILE=true
```

Comment:
But we're not checking here if `SKIP_AUTO_COMPILE` was true or not, so we would effectively always run this command, only to rebuild everything with common_test (?)

Reply:
yes, because there are two scripts. travis-build-tests and travis-tests you can call separately.

But for travis-test to work with SKIP_AUTO_COMPILE, we need this patch of code to be here. So, once SKIP_AUTO_COMPILE stops working, `git grep SKIP_AUTO_COMPILE` would lead here for easy debugging.